packages/itwinui-css/src/tree/tree.scss

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import '../utils/index';
 
 $iui-active-outline: thin solid var(--iui-color-foreground-primary);
-$iui-expander-inline-padding: $iui-component-offset + $iui-xxs + 1px; // margin + padding + border
-$iui-expander-button-width: ($iui-icon-default) + ($iui-expander-inline-padding * 2); // icon width + inline padding on both sides
+$iui-expander-inline-padding: calc(
+  var(--iui-size-2xs) + 1px
+); // padding of small borderless button + 1px transparent border
+$iui-expander-button-width: calc(
+  var(--iui-component-height-large) + 2px
+); // width of small borderless button (button is 1:1 ratio)
```

Comment:
The formatting on this could be improved.  Also I think the math is wrong, our indentations are too large now:

**Before:**
![Screen Shot 2022-08-12 at 12 16 03 PM](https://user-images.githubusercontent.com/849817/184399117-74fc47ab-74d5-4759-ae18-7f177c96e5b5.png)

**After:**
![Screen Shot 2022-08-12 at 12 16 38 PM](https://user-images.githubusercontent.com/849817/184399257-55ed0fa0-0393-439f-ae98-9fa076f90cea.png)

Reply:
Fixed. There was a typo (`large` -> `small`).